### ISSUE TITLE
nVidia 1060 GTX 3GB ETC mining issue - HiveOS

### ISSUE THREAD

**Issue by 34GL3s** [trexminer/T-Rex#993]
Hi,

I have an issue with mining ETC on some of my 1060 3GB rigs.
1 rig with the same cards is working but the other rig has stopped showing some "Can't find nonce with device. Not enough memory". The rig configuration including software versions (OS version, nvidia driver, t-rex driver,..) are totally the same. 
is anybody else experiencing similar issue as I am?

**Comment by trexminer** [comment]
Probably Xorg is consuming more VRAM on your first card than on the rest. You can check that by running `nvidia-smi` when the GPUs are idle.

**Comment by trexminer** [comment]
Yes, it looks fine. Perhaps there is enough space for DAG but not for CUDA kernels and GPU#0 was the first to hit this limit. I believe it's time to switch to another coin as DAG keeps growing.

**Comment by 34GL3s** [comment]
Do you think changing GPU#0<->GPU#3 would make any difference?

**Comment by trexminer** [comment]
It shouldn't but I wouldn't bet my house on it :)
Even if it did, it's only a matter of 1-2 weeks before you run out of VRAM because of DAG growth.

**Comment by 34GL3s** [comment]
alright, thank you for your support :)
will try to find another profitable coin.
